app/models/rubygem.rb

```diff
@@ -153,6 +153,7 @@ def payload(version = versions.most_recent, protocol = Gemcutter::PROTOCOL, host
       'info'              => version.info,
       'licenses'          => version.licenses,
       'metadata'          => version.metadata,
+      'yanked'            => version.yanked?,
```

Comment:
Please explain rationale of adding this in the commit message.

Reply:
Done!